gtsam/sam/tests/testRangeFactor.cpp

```diff
@@ -47,25 +47,29 @@ double measurement(10.0);
 /* ************************************************************************* */
 Vector factorError2D(const Pose2& pose, const Point2& point,
     const RangeFactor2D& factor) {
-  return factor.evaluateError(pose, point);
+  const auto &keys = factor.keys();
+  return factor.unwhitenedError({{keys[0], genericValue(pose)}, {keys[1], genericValue(point)}});
```

Comment:
Hmmm. Since in the factor we know all types, is there not a "variadic" way to make an overload that justs accept the pose and point?

Reply:
:+1: Much easier to use. Done too in 0ee5fc58f